docs/administration/data/data-migration.md

```diff
@@ -28,7 +28,7 @@ The data migration tools are not suitable for every imaginable scenario. In thes
 1. For general disaster recovery learn about [backup and restore for your Octopus Server](/docs/administration/data/backup-and-restore.md).
 1. To move your Octopus database to another database server see [moving your database](/docs/administration/managing-infrastructure/moving-your-octopus/move-the-database.md).
 1. To move your Octopus Server and database to another server see [moving your Octopus Server and database](/docs/administration/managing-infrastructure/moving-your-octopus/move-the-database-and-server.md).
-1. To move your entire Octopus Server from a self-hosted installation to Octopus Cloud, contact our [support team](https://octopus.com/support) for help.
+1. To move your entire Octopus Server from a self-hosted installation to Octopus Cloud, see [migrating from self-hosted to Octopus Cloud](/docs/octopus-cloud/migrations.md).
 
```

Comment:
Believe it or not, we just had our first case of "I want to go from Octopus Cloud to self-hosted..." 🤓

Reply:
I'll add it in as a "contact support" type case